### DISCUSSION TITLE
Error Message

### DISCUSSION THREAD

**Discussion by txproot** [SFX-WoW/Masque#228] [Issues] [answered]
### Game Flavor

TBC Classic

### Game Version

2.5.4.42940

### Add-On Version

9.2.0

### Description

Was scrolling my guild list and the error message popped up.

### Error Message

```text
[Error Message] Message: [ADDON_ACTION_FORBIDDEN] AddOn 'Masque' tried to call the protected function 'GuildControlSetRank()'.
Time: Sat Apr 16 14:15:08 2022
Count: 1
Stack: [ADDON_ACTION_FORBIDDEN] AddOn 'Masque' tried to call the protected function 'GuildControlSetRank()'.
[string "@Interface_TBC\FrameXML\FriendsFrame.lua"]:2526: in function `GuildControlPopupFrame_Initialize'
[string "@Interface_TBC\FrameXML\FriendsFrame.lua"]:978: in function `FriendsFrame_OnEvent'
[string "*:OnEvent"]:1: in function <[string "*:OnEvent"]:1>

Locals: <none>
```

**Comment by StormFX** [comment, marked as answer]
This isn't an error in Masque, as Masque doesn't call that function. Another add-on is throwing the error and it's being propagated by shared libraries. All you can do is disable all of your add-ons and then enable them one-by-one until the error returns to find out which one is causing it.
